### ISSUE TITLE
Cuda environment

### ISSUE THREAD

**Issue by LiaoChihWei** [mulns/AccFlow#5]
Hello, thanks for your great work!

I am interested in trying your model on my team's project.
But I am suffering from some requirement.txt error. I think is cuda version and python version mismatch.
Could you share your AccFlow's conda environment information? 

Your help and advice will be very helpful. Thanks in advance.
@mulns

**Comment by sunz123** [comment]
Hello, have you adjusted this environment?

**Comment by sunz123** [comment]
> Hello, thanks for your great work!
> 
> I am interested in trying your model on my team's project. But I am suffering from some requirement.txt error. I think is cuda version and python version mismatch. Could you share your AccFlow's conda environment information?
> 
> Your help and advice will be very helpful. Thanks in advance. @mulns
Hello, have you adjusted this environment?

**Comment by sunz123** [comment]
> Hello, thanks for your great work!
> 
> I am interested in trying your model on my team's project. But I am suffering from some requirement.txt error. I think is cuda version and python version mismatch. Could you share your AccFlow's conda environment information?
> 
> Your help and advice will be very helpful. Thanks in advance. @mulns

Hello, have you adjusted this environment?

**Comment by LiaoChihWei** [comment]
No, I haven't adjusted this environment. Do you have any suggestion? 
Thanks in advance.

**Comment by mulns** [comment]
Sorry for the delay, I've updated the environment file, please try running`conda env create -f environment.yml` again, let me know if any trouble occurs!

**Comment by LiaoChihWei** [comment]
Thanks for your response. I am trying new environment file. I wonder know whether CUDA version is 12.1?
Because I have PackagesNotFoundError, I am not sure if my environment is not support this.

![螢幕擷取畫面 2024-01-19 235424](https://github.com/mulns/AccFlow/assets/118074893/23d3ce99-4cb2-47ae-a63a-0c96e2b23367)

I just started learning english, please guide me along. 
Thanks for your help!

**Comment by mulns** [comment]
Please show me your command and the full error information

**Comment by LiaoChihWei** [comment]
Thanks for your help! I successfully create the environment. 
I am trying use your model to calculate the optical flow, but results is not reasonable. 
I check is your test_cvo's image tensor size is [5,3,512,512]. I don't know why first order is 5 because my tensor size's first order is 1. 

I print this part:
![image](https://github.com/mulns/AccFlow/assets/118074893/821479d5-98d1-4c3d-b9cf-1a804c11bed2)
How can I preprocess my image before use AccFlow to estimate optical flow?

Thanks for your help!

**Comment by jiangxiaogg** [comment]
我查看了requirement.txt，这个环境是在linux上安装的环境，请问你们 的这个工程能在windows平台运行吗

**Comment by mulns** [comment]
是可以部署在Windows上的，但是在Windows上运行的调试尚未进行，我可以这周尝试一下。在 2024年6月14日，16:51，jiangxiaogg ***@***.***> 写道：﻿
我查看了requirement.txt，这个环境是在linux上安装的环境，请问你们 的这个工程能在windows平台运行吗
